### ISSUE TITLE
app.py errors

### ISSUE THREAD

**Issue by ptits** [fudan-generative-vision/hallo2#20]
1.
    config = {
...
        'config': 'configs/inference/default.yaml',

There is no default.yaml file in repo, needed to copy long.yaml

2. from inference import inference_process

inference.py was renamed by devs to inference_long.py - makes error

3. inference_process returns wrong type and wrong path
should be 
    return output_file  - not save seg path

**Comment by theoldsong** [comment]
Yes, I have also discovered it. Thank you for your efforts. But after fixing it according to your method, the error still occurred. I hope the author can work together to fix it. Thank you, author.

**Comment by theoldsong** [comment]
I changed 'return save_seg_path' to 'return output_file', but still got an error message. 
PermissionError: [Errno 13] Permission denied: 'D:\\py\\hallo2\\output_long\\debug\\g\\seg_video

**Comment by AricGamma** [comment]
app.py is out-of-dated. We'll fix it  soon.

**Comment by Kallamamran** [comment]
I have this now :(
```
(venv) Q:\hallo2>python scripts\app.py
SupervisionWarnings: BoundingBoxAnnotator is deprecated: `BoundingBoxAnnotator` is deprecated and has been renamed to `BoxAnnotator`. `BoundingBoxAnnotator` will be removed in supervision-0.26.0.
Traceback (most recent call last):
  File "Q:\hallo2\scripts\app.py", line 15, in <module>
    from inference import inference_process
ImportError: cannot import name 'inference_process' from 'inference' (Q:\hallo2\venv\lib\site-packages\inference\__init__.py)
```
Which is strange since I ran this 2 weeks ago without a problem :'(

**Comment by AricGamma** [comment]
#65
